cardverificationsheet/src/main/java/com/stripe/android/cardverificationsheet/cardverifyui/VerifyConfig.kt

```diff
@@ -7,38 +7,44 @@ import com.stripe.android.cardverificationsheet.framework.time.seconds
 object VerifyConfig {
 
     /**
-     * The maximum duration for which to search for a card number once sufficient verification
-     * images have been found.
+     * The duration after which the scan will reset if no card is visible.
      */
     @JvmStatic
-    var PAN_SEARCH_DURATION = 5.seconds
+    var NO_CARD_VISIBLE_DURATION = 5.seconds
```

Comment:
These seem to be constants, why are they vars?
Can you use `const val` in an `object`?

Reply:
These are configurable by the merchant. We have seen merchants with demographics of slower phones where it's beneficial for the merchant to be able to increase these values.